builder/lxd/config.go

```diff
@@ -15,6 +15,10 @@ import (
 
 type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
+	// The lxd client used to connect to the lxd server.
+	// Either cli (uses the lxc CLI) or api (uses the lxd client library).
+	// Defaults to cli. 
+	LXDClient string `mapstructure:"lxd_client" required:"false"`
```

Comment:
It should be automatically added by code generation because of the `//go:generate mapstructure-to-hcl2 -type Config` line at the top of the file, not sure if it's linked now